src/model/subscriber.rs

```diff
@@ -0,0 +1,10 @@
+#[derive(Debug, Clone)]
+pub struct NewSubscriber {
+    pub email: String,
```

Comment:
So I haven't done much with this, but what do you think about adding something like an `Email` struct which is
```rust
pub struct Email(pub String);
```
It would kind of just be a wrapper, but in the future if we wanted to add behaviour to it, we can, and it kinda works as a "yes this param is definitely an email".

Adding serialization/deserialization of that is pretty easy too.